ddht/v5_1/pool.py

```diff
@@ -45,18 +46,23 @@ def __init__(
             events = Events()
         self._events = events
 
-        self._sessions = {}
+        self._sessions = LRU(128, callback=self._evict_session)
```

Comment:
I think this should be a configurable parameter that can be controlled from the CLI.  Something like `--session-cache-size`